scripts/prepare_disk.sh

```diff
@@ -1,10 +1,9 @@
 #!/bin/sh
-#
-# SPDX-License-Identifier: AGPL-3.0+
-#
+
```

Comment:
i can agree with you moving the license identifier below the copyright notices, that's good :+1: 
but keep it as one comment block:
```sh
#shebang
#
#stuff
#
# license :)
```